### ISSUE TITLE
[Usable - Bot Framework Web Chat - Markdown]: Narrator is not announcing the name for the 'https://bing.com?q=some' and 'http://www.markitdown.net/' links

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4716]
### **User Experience:**
Screen reader users get affected and cannot know the purpose of the links if the screen reader doesn't announce the name of the link when it receives focus. Screen reader users cannot understand its purpose, they miss accessing it.
### **Note:** 
User credentials should NOT be included in the bug.
### **Repro Steps:**
1. Launch URL: https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/ on edge dev.
2. Web chat page opens.
3. Press 4-tab keys to reach 'Type your message' field. Enter the "markdown" keyword on it. Press 1-tab key to reach the 'send' button and activate it with the enter key.
4. Press 2-shift tabs and 1-down arrow to reach the received card. Activate it with enter key. Focus lands on 'mark it down' link. Turn on screen reader (Narrator).
5. Press 1 or 2 tab keys to reach 'https://bing.com?q=some' and 'http://www.markitdown.net/ links'.
6. Verify whether the screen reader announces the name of the links or not.
### **Actual Result:**
Narrator is not announcing the name for the 'https://bing.com?q=some' and 'http://www.markitdown.net/' links
**Observation:**

- Narrator announces as "link" for both the links when focus lands.
- NVDA Behavior: It announces as "https://bing.com?q=some](https://bing.com/?q=some) Opens in a new window; external. link Opens in a new window; external" and " http://www.markitdown.net/ Opens in a new window; external. link Opens in a new window; external."

- JAWS Behavior: JAWS announces as "https://bing.com?q=some](https://bing.com/?q=some) Opens in a new window; external. Link" and "`http://www.markitdown.net/ Opens in a new window; external. Link

### **Expected Result:**
Narrator should announce the name for the 'https://bing.com?q=some' and 'http://www.markitdown.net/' links when focus lands on them.
### **Note:**
- Issue doesn't repro with JAWS and NVDA.
### Reference Links:

- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDV8MDF8fDNkZjhhMWU0NTA0YTQwM2MyNGRhMDhkYWY1NTkwMDNlfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgwOTIwNTk4MzEzMTM2MjZ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxNVFkzTXpZd09URTRNall3TXpzeE5qY3pOakE1TVRneU5qQXpPekU1T2pjMllUWXlNRE5pTnpZeFpqUmlObVE0T1dKbE0yRm1aVFZpTjJZd1pHSmhRSFJvY21WaFpDNTJNZz09fDBiZDAxMWU1M2M0YjQ0NGYyNGRhMDhkYWY1NTkwMDNlfGI0NDRlYzhjODQ4YzRiODI4Mzk1NmRjNDdlZjQ3NDky&sdata=dlJVM3lpMWQrRmtpc09HVTI0REh4NlVrU00yMG5xd0JlUFdBOHZuaHNOVT0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-pgopinadhu%40microsoft.com&OR=Teams-HL&CT=1673616420039&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzAxMDUwOTQwMCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D). “HCL Staff should not log any third-party external bugs. Should be routed to edad team”.
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process related queries.
### **Test Environment:** 

- OS: Windows 11 Enterprise 22H2 (OS build: 22621.1555)
- Browser: Edge Version 114.0.1807.6 (Official build) dev (64-bit)
- URL: [Web Chat: Full-featured bundle (microsoft.github.io)](https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/)
- Tools: Developer Tool
- Screen Readers: Narrator, JAWS, NVDA
- Matrix: Edge dev+ Narrator.

**Comment by pgopinadhu** [comment]
#A11yUsable;#A11ySev4;#HCL;#Accessibility;#Benchmark;#Win11-Edge(Chromium);#BotFrameworkWebChat-Apr23

**Comment by compulim** [comment]
This is an issue with the bot content, rather than Web Chat. I am going to fix it on the bot side.

**Comment by compulim** [comment]
This is an issue with Windows Narrator + Edge 116.0.1931.0 and is outside of Web Chat.

It seems when the link starts with `https:` and followed by an `<img>`, Windows Narrator will only narrate "link".

This is the minimal repro.

```html
<!DOCTYPE html>
<html>
  <head> </head>
  <body>
    <p>
      Not repro:
      <a href="https://bing.com/">Hello<img /></a> will be narrated as "link, hello".
    </p>
    <p>
      Repro:
      <a href="https://bing.com/">https:Hello<img /></a> will be narrated as "link".
    </p>
    <p>
      Not repro:
      <a href="https://bing.com/">https:Hello</a> will be narrated as "link, H-T-T-P-S hello".
    </p>
    <p>
      Repro:
      <a href="https://bing.com/">ftp:Hello<img /></a> will be narrated as "link".
    </p>
    <p>
      Repro:
      <a href="https://bing.com/">anything:Hello<img /></a> will be narrated as "link".
    </p>
    <p>
      Not repro:
      <a href="https://bing.com/">anything: Hello<img /></a> will be narrated as "link, anything hello".
    </p>
  </body>
</html>
```
